lib/afterpay/consumer.rb

```diff
@@ -4,11 +4,11 @@ module Afterpay
   class Consumer
     attr_accessor :email, :phone, :first_name, :last_name
 
-    def initialize(email:, phone:, first_name:, last_name:)
-      @email = email
-      @phone = phone
-      @first_name = first_name
-      @last_name = last_name
+    def initialize(attributes = {})
+      @email = attributes[:email]
+      @phone = attributes[:phone] || ""
+      @first_name = attributes[:first_name] || ""
```

Comment:
Shouldn't we use single quotes for string since there is no string interpolation here? Does rubocop not complain about this?

Reply:
@siddyg I was also using single quotes but it gived rubocop:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

Reply:
Oh that's interesting